src/Converter/PasswordConverter.js

```diff
@@ -34,7 +34,11 @@ export default class PasswordConverter extends AbstractConverter {
                 clone.customFields = this._customFieldConverter.fromArray([]);
             }
         } else {
-            clone.customFields = this._customFieldConverter.fromArray([]);
+            if(Array.isArray(clone.customFields) !== undefined && clone.customFields.length > 0) {
```

Comment:
Isn't the return value of Array.isArray a boolean?

Reply:
of course you are right, some stupid error from my side, I fixed right now